modules/gridBidAdapter.js

```diff
@@ -134,9 +140,24 @@ function _addBidResponse(serverBid, bidsMap, bidResponses) {
           currency: 'USD',
           netRevenue: false,
           ttl: TIME_TO_LIVE,
-          ad: serverBid.adm,
           dealId: serverBid.dealid
         };
+        if (!serverBid.w && !serverBid.h) {
+          bidResponse.vastXml = serverBid.adm;
+          bidResponse.mediaType = VIDEO;
+          bidResponse.adResponse = {
+            content: bidResponse.vastXml
+          };
+          if (!bid.renderer && (!bid.mediaTypes || !bid.mediaTypes.video || bid.mediaTypes.video.context === 'outstream')) {
+            bidResponse.renderer = createRenderer(bidResponse, {
+              id: bid.bidId,
+              url: RENDERER_URL
+            });
+          }
+        } else {
+          bidResponse.ad = serverBid.adm;
+          bidResponse.mediaType = BANNER;
+        }
```

Comment:
Hi this all looks great,

I have a question as to why when it is video there is no `bidResponse.width` or `bidResponse.height` added?

I see it is not sent or returned by your adserver.

You can see in the below screenshot that the KVP that gets set for the size turns out to be `undefinedxundefined`

![image](https://user-images.githubusercontent.com/13972794/52663052-97f94000-2eba-11e9-81b6-739953c8e064.png)

I have a feeling this _may_ cause problems for certain publishers ad servers maybe?

Do you think it is a good idea to set the width and height based on the bidRequest object?

Reply:
According to Prebid Docs:
http://prebid.org/dev-docs/bidder-adaptor.html#interpreting-the-response

![image](https://user-images.githubusercontent.com/13972794/52663449-84020e00-2ebb-11e9-853e-4655ea093a55.png)

Reply:
width and height are required.

Reply:
Hi, thank you for the catch, we have updated code, please review again.